lib/backend/k8s/resources/node.go

```diff
@@ -233,6 +233,9 @@ func K8sNodeToCalico(k8sNode *kapiv1.Node, usePodCIDR bool) (*model.KVPair, erro
 	}
 	bgpSpec.IPv4IPIPTunnelAddr = annotations[nodeBgpIpv4IPIPTunnelAddrAnnotation]
 
+	// Add in an orchestrator reference back to the Kubernetes node name.
```

Comment:
nit:  Might be nice to state that this is the same as the node name in the kdd case